### PR TITLE
Blatant addition of my own magic items

### PR DESCRIPTION
**Pull request by zontan**
Hi Kevin

### CODE REVIEW

**Comment by Vadskye** [comment]
Hmmm. I suspect that this item would have weird play patterns because of the way it interacts with the attunement mechanics. Since it has no passive effect, you would have to decide at the start of each fight whether you would rather have this item attuned or whether you would rather have an attuned item with a helpful passive effect. Once you have used this item in a fight, there's no point in wearing it for the rest of the day, so you would once again swap it for a different item (assuming you are still conscious) for the rest of the day. It all functions in the rules, but I think the play pattern would be unintuitive, and staying attuned to this item at all times would be a bit of a noob trap.

Also, the "until you take a short rest" wording is dangerous for *beneficial* effects, because it creates an incentive to avoid resting. There is no mechanic requiring you to rest, so you could easily activate this item and then maintain its benefit for the next eight hours if you needed to. That's not helpful for fights, but it is great if you gained fatigue in another context, such as exerting yourself during overland travel. As an 11th level item, that isn't necessarily overpowered, but it is again pretty unintuitive.

Basically, long cooldowns and attuned items don't really mix. I think a mechanic like this wants to be attached to a potion instead, probably. Does that make sense?

**Comment by Vadskye** [comment]
One thing that could help would be to replace the "you can't use this again" cooldown with "increase your fatigue level by 1 when this wears off", which is thematically appropriate and doesn't create the same weird incentives to take off the fatigue-helping item when you are very fatigued. I don't have a clean wording to replace the "until you rest" oddness, but a 5 minute timed effect might work? That's long enough that no one is ever going to need to measure it in a fight.
